setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.9.1
+current_version = 0.9.2
```

Comment:
@aulemahal you forgot to do update `CHANGES.rst` since you bump the tag here.

Something like this would be good:

```
      0.9.2 (2022-07-19)
      ==================
      * Fix Finch unable to startup in the Docker image.
```

Reply:
I created a PR https://github.com/bird-house/finch/pull/252 to address this for you.  Thanks for fixing the docker startup problem.